src/views/Settings.vue

```diff
@@ -16,6 +16,12 @@
             <ion-select-option v-for="facility in (userProfile ? userProfile.facilities : [])" :key="facility.facilityId" :value="facility.facilityId" >{{ facility.name }}</ion-select-option>
           </ion-select>
         </ion-item>
+        <!-- Select store -->
+        <ion-item>
+          <ion-icon :icon="sendOutline" slot="start" />
+          <ion-label>{{$t("Shipping orders")}}</ion-label>
+          <ion-toggle :checked="shippingOrders" @ionChange="showShippingOrders($event)" />
```

Comment:
Add slot end to this

Reply:
Improved Sir,
Thank you sir for reviewing my code.